src/database.ts

```diff
@@ -1744,13 +1936,18 @@ class Database extends ServiceObject {
    * region_tag:spanner_create_storing_index
    * Creating a storing index:
    */
-  updateSchema(statements, callback) {
+  updateSchema(
+    statements:
+      | Schema
+      | database_admin_client.spanner.admin.database.v1.IUpdateDatabaseDdlRequest,
```

Comment:
`Schema` interface already contains `database_admin_client.spanner.admin.database.v1.IUpdateDatabaseDdlRequest`, right?

Reply:
Yes, we don't need `database_admin_client.spanner.admin.database.v1.IUpdateDatabaseDdlRequest`